### PR TITLE
Renew README

### PR DESCRIPTION
**Pull request by kozlovpn**
Update README.rst

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -6,7 +6,27 @@ Shellper
...
+This is new project from the young guys! This application will help you
+organize your day, remind you of upcoming events, help your to cook dinner or
+small lunch and much more! If you need to do some tasks (from school or
+university, maybe working) just run our app and create event! It will pick up
+the latest information and remind you about this! Very simple! Full
+integration with Google Calendar.
+
+**Functionality**
+
+For now added basic functionality:
```
Comment (ivanovmi) [inline README.rst]: Maybe use "we added" instead "added"?

Comment (esikachev) [inline README.rst]: @ivanovmi I agree

**Comment by ivanovmi** [comment]
+1, IGTM wait for CI

**Comment by esikachev** [comment]
+2

**Pull request merged**